RELEASE.md

```diff
@@ -1,5 +1,6 @@
 ## Releasing a new version of Leaflet
 
+- [ ] Ensure all https://github.com/Leaflet/Leaflet/labels/blocker issues and pull requests are resolved.
```

Comment:
```suggestion
- [ ] Ensure all [blocker](https://github.com/Leaflet/Leaflet/labels/blocker) issues and pull requests are resolved.
```

Reply:
So you think we should only use a link and not the label badge self https://github.com/Leaflet/Leaflet/labels/blocker?

https://github.com/Leaflet/Leaflet/pull/8019#discussion_r812571731

Reply:
The difference here is a "label link" vs normal link as seen in https://github.com/Leaflet/Leaflet/pull/8019#discussion_r812571731. We're going with a normal link then?

Reply:
Ah, I forgot the GitHub does this with label links! Disregard this then.